DEPENDENCIES_MANUAL_INSTALL.md

```diff
@@ -32,7 +32,7 @@ manager:
 ## Elasticsearch
 
 To install Elasticsearch, follow [this helpful guide](https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-elasticsearch-on-ubuntu-16-04) for Linux-based systems or the [official instructions](
-https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html).
+https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html).
```

Comment:
Good catch!
How about this link, specifically for ElasticSearch (we don't use any other item in their stack) and for version 6.3?
https://www.elastic.co/guide/en/elasticsearch/reference/6.3/install-elasticsearch.html

Reply:
Okay, I will update it with the above link.